content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
+date: 2018-03-07
+hidden: true
+image: TODO
+---
+
+# 2018.02 Changes
+
+Vision:
+
+- Completely redo vision to greatly increase its accuracy (now all Cartesian co-ordinates are within 15% of their distance from the camera, all angles are within 5 degrees.)
```

Comment:
the additional sentence on the margin of error doesnt make sense here (semantically). I also think these things should be moved to the FAQ page rather than the announcement page, as they're likely going to be asked after often

Reply:
I agree with this, but it should be mentioned both here and on the FAQ page

Reply:
What's the use of it being mentioned here. The FAQ page (or the vision page) is much better, as the information isnt specifically to do with the release.

Reply:
Fixed in cd4dee4c7b282108d92559f999505367b8dc324f

Reply:
Equally, what's the harm in including it here?

Why make information harder to find for students?